### ISSUE TITLE
Custom plugin not being called

### ISSUE THREAD

**Issue by AlessioMartorelli** [alerta/alerta-contrib#134]
Hi! I'm trying to develop a plugin. This might not be an issue but I've tried copying and pasting existing ones and tweaking them for  what I need but doesn't seem to work.
This is the setup.py file:
https://pastebin.com/085D1Uef

This is my script which basically issues a couple of HTTP requests:
https://pastebin.com/byAqLkpa

I installed the plugin running `python setup.py install` and added it in the `PLUGINS` variable of `/etc/alertad.conf`, along with my custom variable `MYPLUGIN_API_BASE_URI` like this:
```
PLUGINS = ['myplugin']
MYPLUGIN_API_BASE_URI = 'http://myurl.domain.com:81'
```

I have a TICK stack (influxdb) and the alerts are successfully pushed from Kapacitor to Alerta, but seems like my plugin doesn't get called.

Anything I'm overlooking here?
Thanks

**Comment by satterly** [comment]
You're missing a "%s" here ...
```
        message = "%s alert for %s - %s is %s" % (
            alert.environment, alert.severity,
            ','.join(alert.service), alert.resource, alert.event
        )
```
When I ran the `alerta` command-line client it gave me the following error message which helped...
```
$ alerta --profile dev8081admin send -r r1 -e e1 -E Production -S Foo -s warning -t test1 -r r3
2017-08-01 07:54:46,271 - alertaclient.shell - ERROR - Error while running post-receive plug-in 'alerta_myplugin': not all arguments converted during string formatting
```

**Comment by AlessioMartorelli** [comment]
Yes, I definitely overlooked it. The weird thing is that I was running alerta in debug mode and even if I could see the alert showing on the web page I couldn't find anything in the log, hence I suspected it wasn't called at all.
I'll try adding the extra `%s` tonight and see if it fixes it and let you know!
Thanks

**Comment by AlessioMartorelli** [comment]
Thanks @satterly the missing %s was definitely the main issue and after some tweaking I managed to get the whole script to work.
Sorry to open this bug for no reason!
